src/shared-components/dropdown/desktopDropdown.tsx

```diff
@@ -16,27 +16,29 @@ import {
 import { OptionType } from './index';
 
 type DesktopDropdownProps = {
-  value?: string;
-  options: OptionType[];
-  currentOption: OptionType;
-  textAlign: 'left' | 'center';
+  borderRadius: string;
   closeDropdown: () => void;
+  currentOption: OptionType;
+  isOpen: boolean;
   onOptionClick: (event: any) => void;
   onSelectClick: () => void;
-  isOpen: boolean;
+  options: OptionType[];
   optionsContainerMaxHeight: string;
+  textAlign: 'left' | 'center';
+  value?: string;
 };
 
 const DesktopDropdown = ({
```

Comment:
Larger diff is from alphabetizing--only change in this file is adding `borderRadius` (and `shouldBeFullyRounded`) logic.